### ISSUE TITLE
Automate Jinxing an Attack Spell

### ISSUE THREAD

**Issue by sIKE23** [sIKE23/Mage-Wars#350]
@ACG8 - thoughts?

[3/4/2016 1:32:17 PM] Philipp Lindenberg: Is it possible to automate Jinx? The Reveal Prompt comes at the right time. it just needs to prompt only when making an Attack spell ... and cancel that spell like Fumble does it with physical attacks
[3/4/2016 1:32:23 PM] Philipp Lindenberg: (maybe give back the mana)

why only attack spell?

[3/4/2016 1:39:48 PM] Philipp Lindenberg: Because they are automated Chain process
[3/4/2016 1:40:11 PM] Philipp Lindenberg: Enchants or Formulas are not automated, so you can do the Jinx manual too

**Comment by ACG8** [comment]
I gave it a shot, putting it in the cast spell code. The problem is that currently the spell is cast before the player gets a prompt to reveal jinx. So for now we should put it in the avoid attack code. I tried to make a commit, but github is not letting my sync for some reason.

**Comment by sIKE23** [comment]
What is the error that you are getting?

**Comment by sIKE23** [comment]
Any luck on getting your changes committed?

//FC

**Comment by ACG8** [comment]
Yes, I'm not sure what the problem was but it seems to be fine now.

**Comment by nirizr** [comment]
Hi, I commented on the commit referencing this issue. Just wanted to make sure it isn't missed, no pressure.
